lib/logflare/backends/adaptor/clickhouse_adaptor/ingester.ex

```diff
@@ -0,0 +1,201 @@
+defmodule Logflare.Backends.Adaptor.ClickhouseAdaptor.Ingester do
+  @moduledoc """
+  Simplified ingestion-only functionality for ClickHouse.
+  """
+
+  use TypedStruct
+
+  import Bitwise
+  import Logflare.Utils.Guards
+
+  alias Logflare.Backends.Backend
+  alias Logflare.LogEvent
+
+  typedstruct module: Row do
+    field :id, Ecto.UUID.t()
+    field :body, String.t()
+    field :timestamp, DateTime.t()
+  end
```

Comment:
In my understanding this is ported to map the schema that existed previously, but with the new approach doesn't it make more sense to encode LogEvents directly?